### PR TITLE
added live data project

### PR DESCRIPTION
**Pull request by bentley-ritson-umuzi**

### CODE REVIEW

File: content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+
+- To end the game, press the End Game button. (This functionality isn't in the starter code for the first codelab in the series.)
+
+In this codelab, you improve the GuessTheWord app by integrating data binding with LiveData in ViewModel objects. This automates the communication between the views in the layout and the ViewModel objects, and it lets you simplify your code by using LiveData.
+
+
+
+![Title screen](4abf85983013d4f3.png)
+
+![Game screen](4abf85983013d4f3.png)
```
Comment (ry-oc) [inline content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md:47]: game screen and title screen are the same pic?

File: content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+
+![Score screen](d6bf6cf728fb6c09.png)
```
Comment (ry-oc) [inline content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md:49]: this renders as a massive pic

File: content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+})
+```
+
+- Run your app and play the game. Notice that the score in the score fragment is displayed correctly, without an observer in the score fragment.
+
+#### Step 3: Add string formatting with data binding
+
+- In the layout, you can add string formatting along with data binding. In this task, you format the current word to add quotes around it. You also format the score string to prefix Current Score to it, as shown in the following image.
+
+![](b48f4a9fba085e11.png)
```
Comment (ry-oc) [inline content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md:339]: this renders as a massive pic

**Review by ry-oc — CHANGES_REQUESTED** [review]

File: content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md (hunk 1 above)
Comment (bentley-ritson-umuzi) [reply content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md:47]: yes

**Review by ry-oc — APPROVED** [review]

**Pull request closed without merging**